tests/rptest/tests/simple_e2e_test.py

```diff
@@ -98,6 +101,7 @@ def inject_failures():
                                 node=node))
                 time.sleep(5)
 
+        (acks, wc_conf) = (-1, "on") if write_caching else (1, "off")
```

Comment:
Does this test do anything? I don't see the cluster configuration `write_caching` being wired.